eppoclient/configurationstore_test.go

```diff
@@ -21,38 +21,55 @@ const TEST_MAX_SIZE = 10
 
 func Test_GetConfiguration_unknownKey(t *testing.T) {
 	var store = newConfigurationStore(TEST_MAX_SIZE)
-	store.SetConfigurations(dictionary{"randomization_algo": testExp})
-	_, err := store.GetConfiguration("unknown_exp")
+	err := store.SetConfigurations(map[string]experimentConfiguration{
+		"randomization_algo": testExp,
+	})
+
+	assert.NoError(t, err)
+	result, err := store.GetConfiguration("unknown_exp")
```

Comment:
* returning the error object everywhere and testing against it, instead of ignoring